src/main/scala/org/broadinstitute/dsde/agora/server/webservice/util/ServiceMessages.scala

```diff
@@ -89,11 +89,11 @@ object ServiceMessages {
                     entityTypes: Seq[AgoraEntityType.EntityType],
                     username: String)
 
-  case class Query(requestContext: RequestContext,
-                   agoraSearch: AgoraEntity,
-                   agoraProjection: Option[AgoraEntityProjection],
-                   entityTypes: Seq[AgoraEntityType.EntityType],
-                   username: String)
+//  case class Query(requestContext: RequestContext,
+//                   agoraSearch: AgoraEntity,
+//                   agoraProjection: Option[AgoraEntityProjection],
+//                   entityTypes: Seq[AgoraEntityType.EntityType],
+//                   username: String)
```

Comment:
This class will need a lot of cleanup ... tons of stuff never used.

Reply:
Agreed. It looks like we'll be able to remove it altogether.